.github/workflows/ci-on-push.yaml

```diff
@@ -1,6 +1,8 @@
 name: Kata Containers CI
 on:
   pull_request_target:
+    branches:
```

Comment:
Good. On another PR that I reviewed I was about to ask you why not set this property, but I ended up not asking because all this stuffs is already too confusing :)

Reply:
You should have done.  I didn't do simply because I forgot to. :-/